rst_files/introduction_to_types.rst

```diff
@@ -364,17 +376,17 @@ Syntax for Creating Concrete Types
 
 While other sorts of types exist, we almost always use the ``struct`` keyword, which is for creation of composite data types
 
-* "composite" refers to the fact that the data types in question can be used as collection of named fields
+* "Composite" refers to the fact that the data types in question can be used as collection of named fields
 
-* the ``struct`` terminology is used in a number of programming languages to refer to composite data types
+* The ``struct`` terminology is used in a number of programming languages to refer to composite data types
 
 
 Let's start with a trivial example where the ``struct`` we build has fields named ``a, b, c``, are not typed
 
 .. code-block:: julia
 
-    struct FooNotTyped # immutable by default, use `mutable struct` otherwise 
-        a # BAD! Not typed
+    struct FooNotTyped  # Immutable by default, use `mutable struct` otherwise 
+        a               # BAD! Not typed
```

Comment:
@jlperla Just wanted to check about this. The convention we've been using in the rest of the lectures is to avoid explicit type constraints on the fields of an immutable struct (to make them closer to `NamedTuples`s, and to reduce the quantity of type annotations in general).

Reply:
We should *never* have structs without type declartions.

Reply:
It is a performance disaster.  This is part of the reason that we should basically never have structs.

Reply:
Can do
```
struct A{T1, T2, T3}
    x::T1
    y::T2
    z::T3
end
```
might be good to have as the fix.

Reply:
@Nosferican  Do you mean in the current lectures that we haven't been able to rip the structs out of?  Yes, that is the only solution for now.

As for the notes themselves, if you look at http://compare-lectures.quantecon.org/jl/introduction_to_types.html# you will see that is exactly what we show afterwards.

Reply:
Makes sense --- for some reason I figured the compiler would silently parametrize with the field parameters like it does for the `NamedTuple`. 

While we're waiting on build stuff, we can pay extra attention to the `struct` issue.